final_prep/mod_zero_hero.rb

```diff
@@ -74,11 +125,44 @@
 
 # - Create 2 instances of your SuperHero class
 
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  arch_nemisis = "The Syntax Error"
```

Comment:
This is the right idea, though the attribute names referenced in your SuperHero class are missing a piece of syntax. This is needed in your attributes up here, as well is in your methods below.